core/src/main/java/discord4j/core/object/entity/Guild.java

```diff
@@ -286,6 +286,15 @@ public PremiumTier getPremiumTier() {
         return PremiumTier.of(data.getPremiumTier());
     }
 
+    /**
+     * Gets the preferred locale of this guild, only set if guild has the "DISCOVERABLE" feature, defaults to en-US.
+     *
+     * @return The preferred locale of this guild, only set if guild has the "DISCOVERABLE" feature, defaults to en-US.
+     */
+    public String getPreferredLocale() {
+        return data.getPreferredLocale();
```

Comment:
Maybe Guild can return a Locale class?

Reply:
It could be interesting but I don't know if returning a Locale is really 1:1 with Discord API. I will wait for an answer of the D4J admins